src/style.css

```diff
@@ -1,3 +1,97 @@
+html {
+  scroll-behavior: smooth;
+}
+
+/* #region body */
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+/* #endregion */
+
+/* #region header */
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.header_logo {
+  font-size: 0;
+}
+
+/* #endregion */
+
+/* #region nav */
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+nav ul {
+  display: flex;
+  padding: 0;
+}
+
+.nav ul li {
+  margin: 0;
+  padding-right: 20px;
+  text-align: center;
+}
+
+.nav ul li:first-child {
+  margin-left: 0;
+}
+
+.nav ul li:last-child {
+  margin-right: 0;
+  padding-right: 0;
+}
+
+.nav__link {
+  position: relative;
+  text-decoration: none;
+  font-size: 12px;
+  text-transform: uppercase;
+  display: block;
+  height: 60px;
+  line-height: 60px;
+  color: black;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
```

Comment:
Create a variable for repeated color and use it everywhere
https://www.w3schools.com/css/css3_variables.asp